setup.py

```diff
@@ -12,8 +14,7 @@
 
 
 def get_extensions():
-    this_dir = os.path.dirname(os.path.abspath(__file__))
-    extensions_dir = os.path.join(this_dir, "detectron2", "layers", "csrc")
+    extensions_dir = os.path.join("detectron2", "layers", "csrc")
```

Comment:
does this cause any issue? seems working to me locally

Reply:
Aah yes, this caused a problem while using `MANIFEST.in` file and using `include_package_data=True`>.

I used an alternative approach, will revert this back. Good catch, thanks!